src/core/runtime.d

```diff
@@ -509,39 +511,59 @@ Throwable.TraceInfo defaultTraceHandler( void* ptr = null )
 
             override int opApply( scope int delegate(ref size_t, ref const(char[])) dg ) const
             {
-                version( Posix )
+                version(Posix)
                 {
-                    // NOTE: The first 5 frames with the current implementation are
+                    // NOTE: The first 4 frames with the current implementation are
                     //       inside core.runtime and the object code, so eliminate
                     //       these for readability.  The alternative would be to
                     //       exclude the first N frames that are in a list of
                     //       mangled function names.
-                    static enum FIRSTFRAME = 5;
+                    enum FIRSTFRAME = 4;
                 }
-                else version( Windows )
+                else version(Windows)
                 {
                     // NOTE: On Windows, the number of frames to exclude is based on
                     //       whether the exception is user or system-generated, so
                     //       it may be necessary to exclude a list of function names
                     //       instead.
-                    static enum FIRSTFRAME = 0;
+                    enum FIRSTFRAME = 0;
                 }
-                int ret = 0;
 
-                const framelist = backtrace_symbols( callstack.ptr, numframes );
-                scope(exit) free(cast(void*) framelist);
-
-                for( int i = FIRSTFRAME; i < numframes; ++i )
+                version(linux)
                 {
-                    char[4096] fixbuf;
-                    auto buf = framelist[i][0 .. strlen(framelist[i])];
-                    auto pos = cast(size_t)(i - FIRSTFRAME);
-                    buf = fixline( buf, fixbuf );
-                    ret = dg( pos, buf );
-                    if( ret )
-                        break;
+                    import rt.backtrace.dwarf;
```

Comment:
@ibuclaw is right, you can't import an `rt` module from a `core` module. You can use an `externDFunc` binding, like [here](https://github.com/D-Programming-Language/druntime/blob/de5a0bc74c2027afc56d2326130655ad01ad5270/src/object.d#L217).